<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Vofox-Sebastian" name="com.analyzeByScore.Local_Assessment_AnalyzeByTag" tests="27" failures="3" timestamp="20 Nov 2018 05:40:07 GMT" time="336.478" errors="0">
  <testcase name="TCED14301" time="6.030" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14302" time="6.053" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14303" time="0.436" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14304" time="1.286" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14305" time="6.236" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14306" time="6.424" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14307" time="8.672" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14308" time="8.296" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14309" time="11.158" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14310" time="9.684" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14311" time="1.049" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14312" time="6.554" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14313" time="11.393" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14314" time="11.402" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14315" time="41.434" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag">
    <failure type="java.lang.AssertionError" message="Could not find any element matching &#039;//*[@id=&#039;ctl00_MainContent_ddlSchool&#039;]&#039;">
      <![CDATA[java.lang.AssertionError: Could not find any element matching '//*[@id='ctl00_MainContent_ddlSchool']'
at Library.BaseClassOne.find(BaseClassOne.java:226)
at Library.BaseClassOne.select(BaseClassOne.java:461)
at com.analyzeByScore.Local_Assessment_AnalyzeByTag.TCED14315(Local_Assessment_AnalyzeByTag.java:743)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TCED14315 -->
  <testcase name="TCED14316" time="50.284" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag">
    <failure type="java.lang.AssertionError" message="Could not find any element matching &#039;//span[@class=&#039;subheading&#039;]&#039;">
      <![CDATA[java.lang.AssertionError: Could not find any element matching '//span[@class='subheading']'
at Library.BaseClassOne.find(BaseClassOne.java:226)
at Library.BaseClassOne.getText(BaseClassOne.java:396)
at Library.BaseClassOne.getText(BaseClassOne.java:346)
at com.analyzeByScore.Local_Assessment_AnalyzeByTag.TCED14316(Local_Assessment_AnalyzeByTag.java:824)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TCED14316 -->
  <testcase name="TCED14317" time="8.383" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14318" time="1.320" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14319" time="10.271" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14320" time="9.516" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14321" time="8.835" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14322" time="8.821" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14323" time="13.622" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14324" time="14.444" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14325" time="8.337" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14326" time="17.878" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag"/>
  <testcase name="TCED14327" time="48.660" classname="com.analyzeByScore.Local_Assessment_AnalyzeByTag">
    <failure type="java.lang.AssertionError" message="Failed to download document which has extension .PDF expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to download document which has extension .PDF expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at com.analyzeByScore.Local_Assessment_AnalyzeByTag.TCED14327(Local_Assessment_AnalyzeByTag.java:1365)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TCED14327 -->
</testsuite> <!-- com.analyzeByScore.Local_Assessment_AnalyzeByTag -->
